app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -62,6 +74,8 @@ def update
       flash[:error] = error.message
       logger.error "FileSetsController::update rescued #{error.class}\n\t#{error.message}\n #{error.backtrace.join("\n")}\n\n"
       render action: 'edit'
+    rescue WorkflowAuthorizationException
```

Comment:
as in https://github.com/samvera/hyrax/pull/3881/files#r305449920

Reply:
It is. I had problems with this at first and thought it was a bug with 'rescue_from' (which I researched), but it isn't. Removed.